client/analytics/test/lib/load-script/index.js

```diff
@@ -1,7 +1,9 @@
 function fakeLoader( url, callback ) {
 	fakeLoader.urlsLoaded.push( url );
 	if ( callback ) {
-		setTimeout( function() { callback(); }, 0 );
+		setTimeout( function() {
```

Comment:
It's minor, but we could also use `async` lib and call [async.nextTick](https://www.npmjs.com/package/async#nextTick).